java-spring/build.gradle.kts

```
@@ -31,10 +31,10 @@ spotbugs {
 
 dependencies {
     // ditto-java artifact includes the Java API for Ditto
-    implementation("com.ditto:ditto-java:5.0.0-preview.1")
+    implementation("live.ditto:ditto-java:4.12.2-rc.2")
 
     // This will include binaries for all the supported platforms and architectures
-    implementation("com.ditto:ditto-binaries:5.0.0-preview.1")
+    implementation("live.ditto:ditto-binaries:4.12.2-rc.2")
```

The java-spring quickstart app only works with v5.

```suggestion
    implementation("live.ditto:ditto-binaries:5.0.0-preview.1")
```